### PR TITLE
Adding security docs

### PR DESCRIPTION
**Pull request by savitharaghunathan**
Adding basic process for reporting security issues

### CODE REVIEW

File: embargo-policy.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+you are REQUIRED to inform the [security contacts](https://github.com/konveyor/security/SECURITY_CONTACTS.md) of exactly what
+information leaked and to whom. A retrospective will take place after the leak
+so we can assess how to not make this mistake in the future.
+
+Violation of this policy will result in the immediate removal and subsequent
+replacement of you from this list or the Security Contacts.
+
+## Disclosure Timeline
+
+This project sustains a **60 days disclosure timeline** to ensure we provide a
```
Comment (savitharaghunathan) [inline embargo-policy.md:27]: @jwmatthews : do you think 60 days disclosure timeline is acceptable?

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+* [konveyor slack channel](https://kubernetes.slack.com/archives/CR85S82A2)
+* [Mailing list](https://groups.google.com/u/0/g/konveyor-community)
+
+
+## Reporting a vulnerability
+
+Please use the below process to report a vulnerability to the project:
+
+1. Email the konveyor-security@googlegroups.com
```
Comment (savitharaghunathan) [inline SECURITY.md:15]: todo: create private ML

Comment (jwmatthews) [reply SECURITY.md:15]: Created ML and invited folks

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+   acknowledging receipt of the email.
+3. You may be contacted by a member of Konveyor Maintainers to further discuss the reported item.
+   Please bear with us as we seek to understand the breadth and scope of the
+   reported problem, recreate it, and confirm if there is a vulnerability
+   present.
+4. As the issue passes through various stages like triage, code fix, 
+   release planning, the reporter will be kept informed. The reporter will be given credits during the disclosure.
+
+
+## Supported Versions
```
Comment (jwmatthews) [inline SECURITY.md:32]: Is 'supported versions' appropriate for us?

Is this where we would communicate longer lived versions that folks may want to file issues on and see resolved?

File: embargo-policy.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+# Embargo Policy
+
+This policy forbids members of this project's security contacts <!-- TODO: $LINK --> and others
```
Comment (jwmatthews) [inline embargo-policy.md:3]: Do you want to keep the 'TODO'?

File: embargo-policy.md (hunk 1 above)
Comment (jwmatthews) [reply embargo-policy.md:27]: If I understand, this means that within 60 days of being informed we will aim to address the issue and release a fix, along with making the issue public?  That sounds reasonable to me.

**Review by jwmatthews — APPROVED** [review]
LGTM

File: SECURITY.md (hunk 3 above)
Comment (savitharaghunathan) [reply SECURITY.md:32]: I am hoping we have the supported versions in main tackle project and hyperlink it. That way its just one place to update and maintain. I left it as TODO for now but open to changes. WDYT?

File: embargo-policy.md (hunk 4 above)
Comment (savitharaghunathan) [reply embargo-policy.md:3]: Great catch. I need to update it :)

File: embargo-policy.md (hunk 1 above)
Comment (savitharaghunathan) [reply embargo-policy.md:27]: Yes. if we need more time to fix, we have a provision in the policy for communication and extend the timeline.

File: SECURITY.md (hunk 2 above)
Comment (savitharaghunathan) [reply SECURITY.md:15]: Thank you

File: SECURITY.md (hunk 3 above)
Comment (savitharaghunathan) [reply SECURITY.md:32]: removed the table. Now, we just just hyperlink from the main repo. We definitely need to maintain a matrix of supported versions somewhere.

Comment (jwmatthews) [reply SECURITY.md:32]: You addressed my concern, I was unsure of how we would keep this list current.

**Pull request merged**